5_analyze_data/notebooks/compare_plates/nbconverted/compare_plate1and2.r

```diff
@@ -0,0 +1,95 @@
+suppressPackageStartupMessages(library(dplyr))
+suppressPackageStartupMessages(library(ggplot2))
+
+# Set file paths
+coef_dir <- file.path("..", "linear_model", "results")
+plate_1_file <- file.path(coef_dir, "Plate1", "linear_model_all_cp_features.tsv")
+plate_2_file <- file.path(coef_dir, "Plate2", "linear_model_all_cp_features.tsv")
+
+output_file <- file.path("figures", "plate1_vs_2_coefficients.png")
+
+# Load coefficients
+plate_1_df <- readr::read_tsv(
+    plate_1_file, col_types=readr::cols(.default = "d", feature="c")
+)
+
+# Recode column names
+colnames(plate_1_df)[seq(2, ncol(plate_1_df))] <- (
+    paste0(colnames(plate_1_df)[seq(2, ncol(plate_1_df))], "_plate1")
+)
+
+plate_2_df <- readr::read_tsv(
+    plate_2_file, col_types=readr::cols(.default = "d", feature="c")
+)
+
+colnames(plate_2_df)[seq(2, ncol(plate_2_df))] <- (
+    paste0(colnames(plate_2_df)[seq(2, ncol(plate_2_df))], "_plate2")
+)
+
+plate_df <- plate_1_df %>%
```

Comment:
Consider adding a comment to explain what this chunk of code does

Reply:
Since @gwaybio created this code for this PR, I am not 100% sure what it actually does (still learning R). I will refrain from adding a comment here at this time, but I agree that it would be good to have it here.